File: lib-mvpbase-appcompat/src/main/java/com/baozi/mvp/view/LoadView.java
package com.baozi.mvp.view;

import com.baozi.mvp.templet.weight.LoadingPager;

/**
 * Created by baozi on 2017/12/8.
 */

public interface LoadView extends BaseView {
    void showSuccess();

    void showEmpty();

    void showError(Throwable throwable, boolean isShowError);

    void showError(Throwable throwable);

    void showLoading();

    void triggerInit();

    LoadingPager getLoadPager();


}

File: lib-mvpbase-kotlin/src/main/java/com/linfeng/mvp/view/LoadView.kt
package com.linfeng.mvp.view

import com.linfeng.mvp.templet.weight.LoadingPager


/**
 * Created by baozi on 2017/12/8.
 */

interface LoadView : BaseView {

    val loadPager: LoadingPager
    fun showSuccess()

    fun showEmpty()

    fun showError(throwable: Throwable, isShowError: Boolean)

    fun showError(throwable: Throwable)

    fun showLoading()

    fun triggerInit()


}
